evaluation/privacy/asv/asv_train/libri_prepare.py

```diff
@@ -88,6 +88,7 @@ def prepare_libri(
     save_csv_train = save_folder / TRAIN_CSV
     save_csv_dev = save_folder / DEV_CSV
 
+    conf = locals()
     # Check if this phase is already done (if so, skip it)
     if skip(splits, save_folder, locals()):
```

Comment:
use conf here instead of `locals()` (3rd argument of the `skip` function)

Reply:
it's the same, so I didn't change